packages/sdk/src/conversation/interface.ts

```diff
@@ -261,6 +306,60 @@ export interface TeamsFxAdaptiveCardActionHandler {
   handleActionInvoked(context: TurnContext, actionData: any): Promise<InvokeResponse>;
 }
 
+/**
+ * Interface for SSO configuration for Bot SSO
+ */
+export interface SsoConfig {
+  /**
+   * Custom sso execution activity handler class which should implement the interface {@link SsoExecutionActivityHandler}. If not provided, it will use {@link DefaultSsoExecutionActivityHandler} by default
+   */
+  CustomSsoExecutionActivityHandler?: new (ssoConfig: SsoConfig) => SsoExecutionActivityHandler;
+
+  /**
+   * The list of scopes for which the token will have access, if not provided, it will use graph permission ["User.Read"] by default
+   */
+  scopes?: string[];
+
+  /**
+   * Conversation state for sso command bot, if not provided, it will use internal memory storage to create a new one.
+   */
+  conversationState?: ConversationState;
+
+  /**
+   * User state for sso command bot, if not provided, it will use internal memory storage to create a new one.
+   */
+  userState?: UserState;
+
+  /**
+   * Used by {@link SsoExecutionDialog} to remove duplicated messages, if not provided, it will use internal memory storage
+   */
+  dedupStorage?: Storage;
+
+  /**
+   * Settings used to configure an teams sso prompt instance.
+   */
+  ssoPromptConfig?: {
+    /**
+     * Number of milliseconds the prompt will wait for the user to authenticate.
+     * Defaults to a value `900,000` (15 minutes.)
+     */
+    timeout?: number;
+
+    /**
+     * Value indicating whether the TeamsBotSsoPrompt should end upon receiving an
+     * invalid message.  Generally the TeamsBotSsoPrompt will end the auth flow when receives user
+     * message not related to the auth flow. Setting the flag to false ignores the user's message instead.
+     * Defaults to value `true`
+     */
+    endOnInvalidMessage?: boolean;
+  };
+
+  /**
+   * teamsfx configuration for sso
+   */
+  teamsFxConfig?: Partial<AuthenticationConfiguration>;
```

Comment:
seems the AuthenticationConfiguration already a partial interface?

Reply:
Oh, thanks, will update